hack/update-codegen.sh

```diff
@@ -21,16 +21,19 @@ set -o pipefail
 SCRIPT_ROOT=$(dirname ${BASH_SOURCE})/..
 CODEGEN_PKG=${CODEGEN_PKG:-$(cd ${SCRIPT_ROOT}; ls -d -1 ./vendor/k8s.io/code-generator 2>/dev/null || echo ../code-generator)}
 
-# optionally verify
-verify="${VERIFY:-}"
+echo "No code to generate"
```

Comment:
We don't generate our own code at this point.  Not ripping out the plumbing, in case we need to again down the road.

Reply:
Wouldn't it be easy to add back if we ever need it? I'd just remove the script

Reply:
I can remove the script & remove the commands from the makefile, etc.
Personally, I tend to prefer some paper trail for this kind of thing, it already has some of "magical" feel.